apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetManifest.java

```diff
@@ -814,50 +787,107 @@ public void setMbiStart(String value) {
     }
 
     /**
-     * Determines the validity of the SyntheaEndStateProperties elements. Much of the detection
-     * centers on checking for negative values, which all Synthea-generated data will have for
-     * things like bene_id, clm_id, etc.
+     * Gets the value of the beneIdEnd property.
+     *
+     * @return value {@link long }
+     */
+    public long getBeneIdEnd() {
+      return beneIdEnd;
+    }
+
+    /**
+     * Sets the value of the beneIdEnd property.
+     *
+     * @param value {@link long} to set
+     */
+    public void setBeneIdEnd(long value) {
+      this.beneIdEnd = value;
+    }
+
+    /**
+     * Gets the value of the clmIdEnd property.
      *
-     * @return possible object is {@link boolean }
+     * @return value {@link long }
      */
-    boolean isValid() {
-      return (beneIdStart < 0
-          && beneIdEnd < 0
-          && clmGrpIdStart < 0
-          && pdeIdStart < 0
-          && carrClmCntlNumStart < 0
-          && fiDocCntlNumStart < 0
-          && clmIdStart < 0
-          && hicnStart != null
-          && mbiStart != null);
+    public long getClmIdEnd() {
+      return clmIdEnd;
+    }
+
+    /**
+     * Sets the value of the clmIdEnd property.
```

Comment:
Not a huge deal, but I had been writing getters and setters as "Sets the {@link #clmIdEnd)."

Reply:
done.